it/stubs/PenaltiesStub.scala

```diff
@@ -33,11 +33,15 @@ object PenaltiesStub {
   private val fetchOtherPenalties= (penaltyID: String) =>
     s"/penalties/appeals/multiple-penalties-in-same-period?enrolmentKey=HMRC-MTD-VAT~VRN~123456789&penaltyId=$penaltyID&isLPP=false"
 
-  def successfulGetAppealDataResponse(penaltyId: String, enrolmentKey: String, isLPP: Boolean = false, isAdditional: Boolean = false): StubMapping = {
-    val typeOfPenalty = if(isAdditional) PenaltyTypeEnum.Additional else if(isLPP) PenaltyTypeEnum.Late_Payment else PenaltyTypeEnum.Late_Submission
+def successfulGetAppealDataResponse(penaltyId: String, enrolmentKey: String, isLPP: Boolean = false, isAdditional:Boolean = false): StubMapping = {
+
     val uri = if(isLPP) appealUriForLPP else appealUriForLSP
-    val extraAdditionalParam = if(isLPP) s"&isAdditional=$isAdditional" else ""
-    stubFor(get(urlEqualTo(s"$uri?penaltyId=$penaltyId&enrolmentKey=$enrolmentKey$extraAdditionalParam"))
+    var testURL = s"$uri?penaltyId=$penaltyId&enrolmentKey=$enrolmentKey"
```

Comment:
I would advise against using mutable values, even in tests. Is there any reason why we are using mutable values?

Reply:
Yes the URL gets updated , however I can use val for two separate cases ie -LPPadditional and LPP

Reply:
No I don't think it's needed. Updated with the master changes.